apps/sertra/pages/server/[id].tsx

```diff
@@ -54,10 +63,11 @@ const ServerDetail: React.FC<{ detail_listing: ServerListingDetailData }> = ({
 
       <div className={styles.headerRow}>
         <h1 className={styles.listingTitle}>{detail_listing.name}</h1>
-        <button className={styles.joinServerButton}>
+        <button className={styles.joinServerButton} onClick={togglePopup}>
           <Logo />
           Join Server
         </button>
+        {showPopup ? <LaunchingModManPopup setClose={togglePopup} /> : null}
```

Comment:
Not a big deal but I feel the name `setClose` (sets value to "closed") somewhat conflicts with `togglePopup` (sets value to "open" or "closed").